catalog/transformer/catalog-transformer-overlay/pom.xml

```diff
@@ -26,6 +26,10 @@
     <name>DDF :: Catalog :: Transformer :: Overlay</name>
     <packaging>bundle</packaging>
 
+    <properties>
+        <la4j.version>0.6.0</la4j.version>
```

Comment:
You don't need to define a property for this if it's only used once within the same pom.